content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -68,47 +68,16 @@ The DEKs are encrypted with a key encryption key (KEK) that is stored and manage
 
 With KMS v1, a new DEK is generated for each encryption.
 
-With KMS v2, there are two ways for the API server to generate a DEK.
-Kubernetes defaults to generating a new DEK at API server startup, which is then reused
-for resource encryption. However, if you use KMS v2 _and_ enable the `KMSv2KDF`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/), then
-Kubernetes instead generates a new DEK **per encryption**: the API server uses a
+With KMS v2, a new DEK is generated **per encryption**: the API server uses a
 _key derivation function_ to generate single use data encryption keys from a secret seed
 combined with some random data.
-Whichever approach you configure, the DEK or seed is also rotated whenever the KEK is rotated
+The seed is rotated whenever the KEK is rotated
 (see `Understanding key_id and Key Rotation` section below for more details).
```

Comment:
```suggestion
(see the _Understanding key_id and Key Rotation_ section below for more details).
```
_or_ use a hyperlink, adding `<a id="key-id-and-rotation"></a>` at an appropriate place to create a fragment identifier.